vgscheckout/src/main/java/com/verygoodsecurity/vgscheckout/networking/command/core/CompositeCommand.kt

```diff
@@ -0,0 +1,53 @@
+package com.verygoodsecurity.vgscheckout.networking.command.core
+
+import android.content.Context
+import com.verygoodsecurity.vgscheckout.exception.VGSCheckoutException
+import com.verygoodsecurity.vgscheckout.networking.command.GetOrderDetails
+import com.verygoodsecurity.vgscheckout.networking.command.GetSavedCardsCommand
+
+internal class CompositeCommand(
```

Comment:
@Endorf Do we use this class anywhere?

Reply:
Yes. In a Payment config. I just updated code.